### PR TITLE
Remove certification filter

### PR DESCRIPTION
**Pull request by gilbertocampos**
This PR removes the certification filter

### CODE REVIEW

**Comment by gilbertocampos** [comment]
@pmcvtm 
SHOULD: the certification be removed from the `vw_StaffSearch` as well?

**Comment by pmcvtm** [comment]
> @pmcvtm
> SHOULD: the certification be removed from the `vw_StaffSearch` as well?

Yes, let's deep-remove this. We'll be keeping Certificates / Credentials (in some form) on the Profile.

**Comment by gilbertocampos** [comment]
Updated according to feedback

File: src/API/LeadershipProfileAPI/Data/Models/StaffSearch.cs [hunk 1]
```diff
@@ -8,8 +8,7 @@ public class StaffSearch
         public string MiddleName { get; set; }
         public string LastSurname { get; set; }
         public string FullName { get; set; }
-        public int? YearsOfService { get; set; }
-        public string Certification { get; set; }
+        public decimal? YearsOfService { get; set; }
```
Comment (gilbertocampos) [inline src/API/LeadershipProfileAPI/Data/Models/StaffSearch.cs:11]: This was giving me an error when running tests for some reason the view is returning now a [`numeric`] type. Also made the type change in `List.SearchResult`

**Review by pmcvtm — APPROVED** [review]
Looks good. We will likely want to come back and more directly address the CTEs in the view for removing duplicates, but we'll be in there enough next week to follow-up on it.

**Pull request merged**